src/components/JudgingCard.js

```diff
@@ -44,7 +44,7 @@ const cutString = (string, maxLength) => {
   return `${string.substring(0, cut)}...`
 }
 
-export default ({ imgUrl, title, teamName, description, href = '#!' }) => {
+export default ({ imgUrl, title, teamName, description, href = '#!', buttonLabel }) => {
```

Comment:
optional params should be last

```suggestion
export default ({ imgUrl, title, teamName, description, buttonLabel, href = '#!' }) => {
```